libr/core/cmd_anal.c

```diff
@@ -91,14 +91,25 @@ static const char *help_msg_aar[] = {
 static const char *help_msg_ab[] = {
 	"Usage:", "ab", "",
 	"ab", " [addr]", "show basic block information at given address",
+	"ab.", "", "same as: ab $$",
 	"aba", " [addr]", "analyze esil accesses in basic block (see aea?)",
 	"abb", " [length]", "analyze N bytes and extract basic blocks",
 	"abj", " [addr]", "display basic block information in JSON (alias to afbj)",
+	"abl", "[,qj]", "list all basic blocks",
```

Comment:
This line uses `,` form for table commands, but help bellow mentions `t`. I am fine with either approach showing or not showing the both foms in helps, but in the case of showing only one of them it should be the same here and in `abl` help.

Reply:
There’s a PR from me before the drama that i was trying to get some feedback about this. Basically comma is fine for me as a csv char.. and t is not always available as a subcommand. So inwas trying to discuss about which char to pick here. The inconsistency in this pr about handling comma and t is because the same thing happens in other places in r2.

I have two more PRs for rtable and the big one which would be neat to clarify this syntax decission before going much further on this direction.

Reply:
Thanks, for explaining. Missed that discussion.